libs/output-mapping/src/DeferredTasks/Metadata/SchemaColumnMetadata.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\DeferredTasks\Metadata;
+
+use Keboola\OutputMapping\Mapping\MappingFromConfigurationSchemaColumn;
+use Keboola\StorageApi\Metadata;
+use Keboola\StorageApi\Options\Metadata\TableMetadataUpdateOptions;
+use Keboola\Utils\Sanitizer\ColumnNameSanitizer;
+
+class SchemaColumnMetadata implements MetadataInterface
```

Comment:
Bude to znít zle, ale mohl bys prosím tehle class dopsat unit test?

Další věci z `Keboola\OutputMapping\DeferredTasks\Metadata` to nemaj, historicky. To by se mělo doplnit někdy. Na to klidně založíme další task. Ale tady je to čistě nová classa :)

Reply:
přidal jsem to tady - https://github.com/keboola/platform-libraries/pull/280/commits/349e15e6c98c92839adfa5e96ad26c70e8abfb84